utils/date.h

```diff
@@ -25,7 +25,7 @@
 // SOFTWARE.
 //
 // Our apologies.  When the previous paragraph was written, lowercase had not yet
-// been invented (that woud involve another several millennia of evolution).
+// been invented (that would involve another several millennia of evolution).
```

Comment:
This is not ours, so I opted to keep it as such.

Reply:
reverted.

Reply:
will add this file to the skip list.

Reply:
It was fixed years ago (https://github.com/HowardHinnant/date/commit/a811a20748d5354975a698ea6dc6e814146db08a ) which makes you wonder if we should not re-sync the code.

Reply:
guess the reason is that it is a low-priority working item to do. i will take a quick look to evaluate the risk and workload to update this file with the upstream.